configure.sh

```diff
@@ -206,6 +208,30 @@ verify_git_repository() {
     export GIT_TERMINAL_PROMPT=1
 }
 
+setup_github_webhook() {
+    WEBHOOK_SECRET="${BOOTSTRAP_FLUX_GITHUB_WEBHOOK_SECRET:-ignored}"
+
+    if [[ "${WEBHOOK_SECRET}" != "ignored" ]]; then
+        if [[ "${WEBHOOK_SECRET}" == "generated" ]]; then
+            WEBHOOK_SECRET="$(openssl rand -base64 30)"
+        fi
+
+        export BOOTSTRAP_FLUX_GITHUB_WEBHOOK_SECRET="${WEBHOOK_SECRET}"
+        _log "INFO" "Using GitHub Token '${WEBHOOK_SECRET}' for Flux"
+
+        cp -f  "${PROJECT_DIR}/tmpl/cluster/flux-system" "${PROJECT_DIR}/cluster/apps/"
```

Comment:
```suggestion
        cp -rf  "${PROJECT_DIR}/tmpl/cluster/flux-system" "${PROJECT_DIR}/cluster/apps/"
```

Copying folders needs the recursive flag